Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -225,7 +223,7 @@ function Add-SandboxCredentials {
         [string] $populatedSecret = (Get-RandomString),
         [string] $minimalKey = (Get-RandomString),
         [string] $minimalSecret = (Get-RandomString),
-        [string] $adminCredentialConfigPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'Application\EdFi.Ods.SandboxAdmin.Web\AdminCredential.config'),
+        [string] $adminCredentialConfigPath = (Join-Path (Get-RepositoryRoot $implementationRepo) 'Application\EdFi.Ods.SandboxAdmin\AdminCredential.config'),
```

Comment:
should we be using the appsettings.user.json that is generated here?

Reply:
This function is no longer used and will be removed on the next release.